src/lib/utils.ts

```diff
@@ -33,3 +33,18 @@ export function convertTo12HourFormat(time24: string): string {
   // Return the formatted time string
   return `${hours}:${formattedMinutes}${period}`;
 }
+
+export const validateDate = (date: string) => {
+  // format DD/MM/YY
+  const dateParts: string[] = date.split("/");
+  //simple date validation
+  if (dateParts.length != 3 || dateParts.includes("")) return false;
+  if (2 < dateParts[0].length) return false; //DD
+  if (2 < dateParts[1].length) return false; //MM
+  if (dateParts[2].length != 4) return false; //YY
+
+  // Date("year/month/day"),
+  const dateStr = +dateParts[2] + "/" + +dateParts[1] + "/" + +dateParts[0];
+  const dateObject = new Date(dateStr);
+  return dateObject.toString() != "Invalid Date";
+};
```

Comment:
consider using zod to validate this instead
https://zod.dev/?id=dates

https://github.com/react-hook-form/resolvers?tab=readme-ov-file#zod